src/data/finset/lattice.lean

```diff
@@ -1173,6 +1173,28 @@ lemma exists_min_image (s : finset β) (f : β → α) (h : s.nonempty) :
 @exists_max_image αᵒᵈ β _ s f h
 
 end exists_max_min
+
+lemma is_least_of_is_glb [linear_order α] {i : α} (s : finset α)
+  (his : is_glb (s : set α) i) (hs : s.nonempty) : is_least ↑s i :=
```

Comment:
Can you make this an `iff`?
```suggestion
  (hs : s.nonempty) : is_glb (s : set α) i ↔ is_least ↑s i :=
```
We already have `is_least.is_glb` for the reverse direction